mlir_graphblas/src/lib/GraphBLAS/GraphBLASLowerPass.cpp

```diff
@@ -1854,6 +1849,42 @@ class LowerCommentRewrite : public OpRewritePattern<graphblas::CommentOp> {
   };
 };
 
+class LowerVectorArgMinOpRewrite : public OpRewritePattern<graphblas::VectorArgMinOp> {
+public:
+  using OpRewritePattern<graphblas::VectorArgMinOp>::OpRewritePattern;
+  LogicalResult matchAndRewrite(graphblas::VectorArgMinOp op, PatternRewriter &rewriter) const override {
+    Location loc = op->getLoc();
+
+    Value inputTensor = op.vec();
+    Type outputType = op->getResultTypes()[0];
+
+    graphblas::VectorArgMinMaxOp newVectorArgMinMaxOp =
+      rewriter.create<graphblas::VectorArgMinMaxOp>(loc, outputType, inputTensor, "min");
+
+    rewriter.replaceOp(op, newVectorArgMinMaxOp.getResult());
```

Comment:
If you specify the type of `newVectorArgMinMaxOp` as `Value`, you don't need to call `.getResult()`. The conversion to `Value` will happen automatically. I think that's the preferred approach in other lowing methods.

Reply:
I implemented this fix.